### PR TITLE
Bug 1116803 - [rtl] [contacts] when adding new contact the add field and remove field buttons are not coresponding to select +autoland

### PR DESCRIPTION
**Pull request by anefzaoui**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b21502df2f3e)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=864beb220590)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=22218e9a0d90)

File: apps/communications/contacts/style/app.css [hunk 1]
```diff
@@ -637,4 +603,49 @@ html[dir="rtl"] ::-moz-placeholder {
...
+}
+
+html[dir="rtl"] .fillflow-media p {
+  left: unset;
+  right: -1.5rem;
+}
+
+html[dir="rtl"] .contact-checkbox {
+  left: unset;
+  right: 0 !important;
```
Comment (arcturus) [inline apps/communications/contacts/style/app.css]: Can we get ride of this?

Comment (anefzaoui) [inline apps/communications/contacts/style/app.css]: Unfortunately we can't :( Not adding !important will still make the original property a higher priority, thus it's as if we didn't fix it with this rule.

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=604d1c6ea233)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f5a7f9c8bfa9)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=421e6ea49bd1)

**Pull request merged**